### PR TITLE
Adds new fields for metadata

### PR DESCRIPTION
**Pull request by chentex**
## Type of change

- [ ] Refactor
- [x] New feature
- [ ] Bug fix
- [ ] Optimization
- [ ] Documentation Update

## Description

Adds new fields for metadata
- fips
- ipsec
- encryption
- encryption type
- publish

## Related Tickets & Documents

- Related Issue #
- Closes #

## Checklist before requesting a review

- [ ] I have performed a self-review of my code.
- [ ] If it is a core feature, I have added thorough tests.

## Testing
- Please describe the System Under Test.
- Please provide detailed steps to perform tests related to this code change.
- How were the fix/results from this change verified? Please provide relevant screenshots or results.

### CODE REVIEW

File: utils/index.sh [hunk 1]
```diff
@@ -141,6 +153,8 @@ index_task(){
         "encrypted":"'"$encrypted"'",
         "encryptionType":"'"$encryption"'",
         "publish":"'"$publish"'",
+        "computeArch":"'"$compute_arch"'",
+        "controlPlaneArch":"'"$control_plane_arch"'",
```
Comment (afcollins) [inline utils/index.sh:157]: Nice!

**Review by afcollins — APPROVED** [review]
Much appreciated. Thank you! LGTM

**Comment by vishnuchalla** [comment]
@chentex As these changes go into our CI and just to make sure nothing breaks, did we make sure that these changes are working as expected in our local testing?

**Comment by chentex** [comment]
> @chentex As these changes go into our CI and just to make sure nothing breaks, did we make sure that these changes are working as expected in our local testing?

They are going to the CI, I tested all the commands against clusters that had and didn't have the configurations, I am hoping I covered all the scenarios in my local testing

**Review by vishnuchalla — APPROVED** [review]
lgtm

**Comment by jtaleric** [comment]
just putting my .02 here... I hope to one day remove this script and use some sort of cli wrapper on-top of `go-commons` to generate our metadata from the same thing our tools use.

**Comment by chentex** [comment]
Working on adding them [here](https://github.com/cloud-bulldozer/go-commons/pull/40)

**Review by jtaleric — APPROVED** [review]
lets get this in until go-commons can get us a CLI ;)

**Pull request merged**